public/app/models/query/Condition.js

```diff
@@ -0,0 +1,84 @@
+// Based on http://kindohm.github.io/knockout-query-builder/js/condition.js
+(function () {
+    'use strict';
+
+    define(
+        [
+            'lodash',
+            'knockout'
+        ],
+        function (_, ko) {
+            return function (queryBuilder) {
```

Comment:
The following 3 lines define the observables corresponding to the 3 components of the search - field, comparator and value.